### DISCUSSION TITLE
Apply pattern to part of the bar in a bargroup chart.

### DISCUSSION THREAD

**Discussion by brownjedi** [airbnb/visx#1467] [Q&A]
Hi,

I'm using the `visx/xychart` package. I'm trying to create a bargroup chart which shows the difference in a pattern. I was initially thinking of doing a BarStack/BarChart combo but it looks like that's not possible. Is there a way I can achieve this?

Any help would be grateful.

![Screen Shot 2022-04-05 at 8 15 28 PM](https://user-images.githubusercontent.com/6647593/161871411-40391490-450e-4010-8ea1-ea597e1c90f5.png)

**Comment by williaster** [comment]
Hey @brownjedi 👋  beautiful chart! Since `xychart` doesn't currently support groups + stacks (we should look into adding this..), my only thought is to render 2x grouped bar series with the pattern series behind the first group? Will keep noodling on it.
